packages/squiggle-lang/src/public/SqValue/SqPlot.ts

```diff
@@ -222,8 +244,12 @@ export class SqDistFnPlot extends SqAbstractPlot<"distFn"> {
     return wrapScale(this._value.distXScale);
   }
 
-  get xPoints(): number[] | undefined {
-    return this._value.xPoints;
+  xPoints(params: {
+    min?: number;
+    max?: number;
+    extraPoints?: number[];
```

Comment:
This method is a bit confusing; it takes `extraPoints` but in most cases `extraPoints` is the _main_ list of equidistant points generated by `components`, which does `rangeByCount`.

I'd recommend either moving `rangeByCount` logic here, or rename `extraPoints` or the method name somehow.

Reply:
I wanted to move ``rangeByCount`` here, but it uses D3 to choose which X points to take. I assumed we didn't want to load d3 here, and it would be annoying to do these methods without D3 (maybe a future story), but wasn't sure.

Reply:
Any thoughts on what to name ``extraPoints``? From the perspective of squiggle-lang, there are new points. Maybe ``requestedXPoints``?

Reply:
BTW: I changed the name to ``requestedXPoints``, and also changed the components code to always add the requested amount of X points, instead of only adding the diff.